pyaedt/__init__.py

```diff
@@ -1,6 +1,18 @@
 import os
 # Import exception handling here due to:
 # https://github.com/pyansys/PyAEDT/pull/243
+import sys
+
+is_ironpython = False
+if "IronPython" in sys.version or ".NETFramework" in sys.version:
+    is_ironpython = True
```

Comment:
This is a bit cleaner:

```suggestion
is_ironpython = "IronPython" in sys.version or ".NETFramework" in sys.version
```